### DISCUSSION TITLE
Display box for commenting *after* all the comments

### DISCUSSION THREAD

**Discussion by dmerejkowsky** [posativ/isso#785] [Q&A]
I believe it's a good idea for people to at least _glance_ at all the comments before replying, but I may be wrong.

Anyway, I managed to find a quick and dirty solution here:
https://github.com/dmerejkowsky/isso/commit/0da5aa225956a611712524866344845327bdd4bc

But I'm wondering if:
* there's a better way to do it
* it should be configurable
* I could achieve the same thing using CSS

Just curious, I'm happy with my hack for now :)

**Comment by madranet** [comment]
Nearly ten months and no response? ...apart from referencing this issue from my other similar one.

**Comment by ix5** [comment]
> I could achieve the same thing using CSS

Yup, you can set containers as `display: flex` and re-order them at will.

> it should be configurable

Isso is already quite configurable and at some point, all the available options will simply overwhelm even seasoned admins.

**Comment by ix5** [comment]
This is something that can be set via CSS. I remain convinced introducing _yet another_ config option for this would further complicate Isso and make maintenance harder.
